code/game/machinery/slotmachine.dm

```diff
@@ -25,30 +25,42 @@
 	circuit = /obj/item/circuitboard/computer/slot_machine
 	light_color = LIGHT_COLOR_BROWN
 	interaction_flags_machine = INTERACT_MACHINE_ALLOW_SILICON|INTERACT_MACHINE_SET_MACHINE // don't need to be literate to play slots
+	tgui_id = "SlotMachine"
 	var/money = 3000 //How much money it has CONSUMED
 	var/plays = 0
 	var/working = FALSE
 	var/balance = 0 //How much money is in the machine, ready to be CONSUMED.
 	var/jackpots = 0
 	var/paymode = HOLOCHIP //toggles between HOLOCHIP/COIN, defined above
 	var/cointype = /obj/item/coin/iron //default cointype
+	/// Icons that can be displayed by the slot machine.
+	var/static/list/icons = list(
+		"lemon" = list("value" = 2, "colour" = "yellow"),
+		"star" = list("value" = 2, "colour" = "yellow"),
+		"bomb" = list("value" = 2, "colour" = "red"),
+		"biohazard" = list("value" = 2, "colour" = "green"),
+		"apple-whole" = list("value" = 2, "colour" = "red"),
+		SEVEN = list("value" = 1, "colour" = "yellow"),
+		"dollar-sign" = list("value" = 2, "colour" = "green")
```

Comment:
Actually any reason why this isn't using FA icon defines? e.g ``FA_ICON_LEMON``

Reply:
didn't know those were a thing